common/daq/can_config.json

```diff
@@ -30,9 +30,27 @@
                             "msg_period":15,
                             "msg_hlp":1,
                             "msg_pgn":1
+                        },
+                        {
+                            "msg_name": "fault_sync_main_module",
```

Comment:
Sounds good, I was concerned about possibly making this interfere with other CAN messages (in case someone uses identical info), but I can add this to the CAN generator, and make them "reserved words". I think that is the best way of hiding them from the fault config. The only concern I had (and still have) with that is the possibility that it will interfere with the DAQ side.

Reply:
All of DAQ is running on the lowest priority. The highest priority is specifically for faults, so feel free to claim that for the library. Note that to make the can messages inserted into the json, you will need to run the generator from the daq/generator.py before the can generator is ran